Cargo.toml

```diff
@@ -21,7 +21,8 @@ default = []
 embedded-hal = "1.0.0-alpha.5"
 nb = "1.0.0"
 
-log = { version = "0.4.14", default_features = false }
+log = { version = "0.4.14", optional = true, default_features = false }
+defmt = { version = "*", optional = true }
```

Comment:
Please no star dependencies. We can use ">=0.2.0,<0.4" if we want to be backwards compatible during the 0.2/0.3 defmt migration.

Reply:
oh good catch, it was just a placeholder that i forgot